app/views/return-requirements/abstraction-period.njk

```diff
@@ -19,7 +17,7 @@
 {% block content %}
   {# Main heading #}
   <div class="govuk-body">
-    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">{{ title }}</h1>
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">Enter the abstraction period for the return requirement</h1>
```

Comment:
if pageTitle will always match the H1 question we should just reuse the pageTitle here rather than duplicating the text

Reply:
Great minds and all that!

![sleep-blocker](https://github.com/DEFRA/water-abstraction-system/assets/1789650/eb91a7ef-7ea1-4ce7-b515-113172edde64)